types/roblox-user-information/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for roblox-user-information 1.0
+// Project: https://github.com/bananaMaster69/roblox-user-information#readme
+// Definitions by: 1chiSensei <https://github.com/1chiSensei>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function getUser(username: string): Promise<object>;
```

Comment:
You might want to consider specifying more specific type instead of the object since the read me shows some fields. But i am not familiar with the library to know if those are always the case or not.